service/utility.go

```diff
@@ -34,7 +36,7 @@ func validateUser(user *pb.User) error {
 	if err := validateEmail(user.GetEmail()); err != nil {
 		return err
 	}
-	if user.GetPassword() == "" {
+	if pass := user.GetPassword(); pass == "" || strings.TrimSpace(pass) != pass {
```

Comment:
I do not agree with the var name `pass` for password, but if you have a reason then I would like to know.